app/models/email_suppression.rb

```diff
@@ -0,0 +1,8 @@
+class EmailSuppression < ApplicationRecord
+
+  ACCOUNT_EMAILS = "account_emails"
+  DONATION_EMAILS = "donation_emails"
+  NEWS_EMAILS = "news_from_inaturalist"
+  TRANSACTIONAL_EMAILS = "transactional_emails"
+
```

Comment:
You might consider adding a validation to ensure that `suppression_type` is one of the accepted values.